src/VisualStudio/LiveShare/Impl/Client/ExternalAccess/VSTypeScript/Api/VSTypeScriptRemoteLanguageServiceWorkspaceAccessor.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio.LanguageServices.ExternalAccess.VSTypeScript.Api;
+
+namespace Microsoft.VisualStudio.LanguageServices.LiveShare.Client.ExternalAccess.VSTypeScript.Api
```

Comment:
💡 I'd move this up one folder to not be in an `Api` folder (the VSTypeScript code is going to be using the interface anyway). Not a blocker but should be an easy change.